helpers/pptx_helper.py

```
@@ -486,6 +489,39 @@ def _handle_display_image__in_background(
+
+                    # Check if alphaModFix already exists to avoid duplicates
+                    existing_alpha_mod = blip.find(
+                        '{http://schemas.openxmlformats.org/drawingml/2006/main}alphaModFix'
+                    )
+                    if existing_alpha_mod is not None:
+                        blip.remove(existing_alpha_mod)
+
+                    blip.append(alpha_mod)
+                    logger.debug('Added transparency to blip element: %s', blip.xml)
+
+            except Exception as ex:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)